### DISCUSSION TITLE
Changing the 'Weekdays to Water' under the Smart Water Restrictions

### DISCUSSION THREAD

**Discussion by morganplambert** [sebr/bhyve-home-assistant#166] [Q&A] [answered]
Looking to create an automation in HA to change the 'Weekdays to Water' under the Smart Water Restrictions during the winter and summer as our restrictions change.  Is this accessable to change? Thanks

**Comment by sebr** [comment, marked as answer]
It's not possible to change the configuration of any B-Hyve program from Home Assistant integration.
